catalog/conformance-tests/result-cirrus-nexus-true-carbon-for-pact-01-002.json

```
@@ -0,0 +1,15 @@
+{
+  "tested_solution": {
+    "solution_id": "cirrus-nexus-true-carbon-for-pact-01",
+    "version": "2.0.0"
+  },
+  "tested_by": {
+    "solution_name": "NRI-CTS",
+    "version": "0.9.15",
+    "solution_id": " nri-cts"
```

There is a small typo: a space before `nri-cts`
```suggestion
    "solution_id": "nri-cts"
```
Thank you!

---

@raimundo-henriques Thanks for pointing out the issue. I have fixed and pushed the latest changes. Please review and let us know

---

Thank you for the changes (and sorry for the delay)! 
It is looking good and we should be able to merge it soon!